packages/element-lib/src/__tests__/Closeable.spec.js

```diff
@@ -0,0 +1,19 @@
+const getLocalSidetree = require('./__fixtures__/getLocalSidetree');
+
+jest.setTimeout(10 * 1000);
+
+let sidetree;
+
+beforeAll(async () => {
+  sidetree = await getLocalSidetree('Closeable');
+});
+
+afterAll(async () => {
+  await sidetree.close();
+});
+
+describe('Closeable', () => {
+  it('works', () => {
+    console.log('asdf');
```

Comment:
mmmh, would this still work if we remove console.log and move sidetree.close here?

Reply:
This should not have been commited, it was part of my jest closing tests..